lib/src/base-styles/_nx-btn.scss

```diff
@@ -110,8 +110,14 @@
 }
 
 .nx-btn--icon-only {
+  border-color: transparent;
   height: 32px;
   padding: 0;
   width: 32px;
   min-width: 0;
+
+  .nx-icon {
+    height: 1em;
+    width: 1em;
```

Comment:
I'm not sure this width is right universally, it might've only been for the X icon

Reply:
> I'm not sure this width is right universally, it might've only been for the X icon

I checked the Zeplin examples and they show icon sizes at 16px so I think that's universal

Reply:
> I checked the Zeplin examples and they show icon sizes at 16px so I think that's universal

But I'm pretty certain that applitools icons aren't all the same aspect ratio.  So if they're all 1em tall they're not (all) also 1em wide

Reply:
> > I checked the Zeplin examples and they show icon sizes at 16px so I think that's universal
> 
> But I'm pretty certain that applitools icons aren't all the same aspect ratio. So if they're all 1em tall they're not (all) also 1em wide

Ah: https://github.com/sonatype/sonatype-react-shared-components/pull/138/commits/253f6db96df88c119282b5409272f37d5c3ac185